thimble-kafka/src/troy_west/thimble/kafka.clj

```diff
@@ -81,9 +82,18 @@
   (.get (.names (.listTopics ^AdminClient (:admin-client broker-state)))))
 
 (defmethod ig/init-key :thimble/kafka.broker
-  [_ {:keys [zookeeper topics tmp-dir config]}]
+  [_ {:keys [zookeeper topics tmp-dir dir config]}]
   (assert zookeeper)
-  (start-broker zookeeper topics tmp-dir config))
+  (start-broker zookeeper topics
+                (if (seq dir)
```

Comment:
Hey Tristan, this all looks good - question: Are you expecting dir to be a sequence here, or a String?

Also maybe we flatten the tmp-dir / dir options into one, but I can do that in a new PR.